### PR TITLE
Adam Stallard Oct 2024 fixes

### PR DESCRIPTION
**Pull request by adamstallard**
Various small fixes to the English content.

### CODE REVIEW

**Review by GlenWeyl — CHANGES_REQUESTED** [review]
These are all good, but you need to resolve conflicting branches.

File: contents/english/3-2-connected-society.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Thus the individual that the national identity systems seek to strip away from t
...
+In his 1927 work that defined his political philosophy, *The Public and its Problems*, John Dewey (whom we meet in [A View from Yushan](https://www.plurality.net/v/chapters/2-1/eng/?mode=dark)) considered the political implications and dynamics of these “emergent publics” as he called them.[^PublicProblems]Dewey's views emerged from a series of debates he held, as leader of the "democratic" wing of the progressive movement after his return from China with left-wing technocrat  Walter Lippmann, whose 1922 book *Public Opinion* Dewey considered "the most effective indictment of democracy as currently conceived".[^Westbrook]  In the debate, Dewey sought to redeem democracy while embracing fully Lippmann's critique of existing institutions as ill-suited to an increasingly complex and dynamic world.
```
Comment (adamstallard) [inline contents/english/3-2-connected-society.md:135]: I resolved the conflict on this line and merged the changes.

**Comment by adamstallard** [comment]
@GlenWeyl ready for you

**Review by GlenWeyl — APPROVED** [review]
Great!

**Comment by GlenWeyl** [comment]
Please join Gov4Git so I can credit you.

**Pull request merged**